VimeoNetworking/Sources/Models/VIMVideo.m

```diff
@@ -436,6 +436,15 @@ - (BOOL)isRatedMature
 
 - (BOOL)isDRMProtected
 {
+    if (self.playRepresentation.drmFiles.fairPlayFile)
```

Comment:
Should we swap consideration of these cases? I remember for VOD, we would sometimes have access to files we shouldn't so we had to look at the play status to determine if it's allowed to be played back. I was wondering if there's a similar edge case here, where checking the stricter of the 2 cases first is worth it.